Libraries/LibJS/Runtime/MathObject.cpp

```diff
@@ -357,4 +366,138 @@ JS_DEFINE_NATIVE_FUNCTION(MathObject::cbrt)
     return Value(::cbrt(number.as_double()));
 }
 
+JS_DEFINE_NATIVE_FUNCTION(MathObject::atan2)
+{
+    auto y = vm.argument(0).to_number(global_object), x = vm.argument(1).to_number(global_object);
+    auto pi_4 = M_PI_2 / 2, three_pi_4 = pi_4 + M_PI_2;
```

Comment:
declaring multiple variables at once is uncommon for this project's coding style, I think.

```suggestion
    auto pi_4 = M_PI_2 / 2;
    auto three_pi_4 = pi_4 + M_PI_2;
```